### DISCUSSION TITLE
Does phase object viscosity impact singlephase permeability calculations?

### DISCUSSION THREAD

**Discussion by taylrcawte** [PMEAL/OpenPNM#1904] [Q&A] [answered]
Hello, i ran the example found [here](https://github.com/PMEAL/OpenPNM/blob/dev/examples/tutorials/intro_to_openpnm_intermediate.ipynb). By replacing the two phase objects with 
```
water = op.phases.Water(...)
air = op.phases.Air(...)
```
and using default values for all pore scale models, i achieved the same results for Kxx regardless of the phase being acted upon in the algorithm object. I.e. 
```
alg = op.algorithms.StokesFlow(network=pn, phase=water)
alg.set_value_BC(values=202650, pores=pn.pores('back'))
alg.set_value_BC(values=101325, pores=pn.pores('front'))
alg.run()
```
and
```
alg = op.algorithms.StokesFlow(network=pn, phase=air)
alg.set_value_BC(values=202650, pores=pn.pores('back'))
alg.set_value_BC(values=101325, pores=pn.pores('front'))
alg.run()
```
yielded the same value for Kxx. 

I was confused by this given the documentation states the StokesFlow algorithm solves Darcy's equaion, which involves the dynamic viscosity of the fluid (air vs water). 

So i am curious to know if i've incorrectly used the logic of openpnm or if this an expected result and why?

**Comment by jgostick** [comment, marked as answer]
Permeability is independent of viscosity. Darcy's law is Q = (K / mu) * (A/L) * DP, so once you calculate Q for a given DP, you then solve for K in Darcy's law since mu, L and A are known.  If the documentations says it's solves Darcy's law, then it must not be very well worded.  I guess technically, the Q is the Darcy flow, but it is the net effect of Stokes flow in each pore.

**Reply by taylrcawte** [reply]
Understood! Thank you!
